### ISSUE TITLE
Style/TernaryParentheses reports false positive

### ISSUE THREAD

**Issue by RST-J** [rubocop-hq/rubocop#7713]
--------

## Expected behavior
`Style/TernaryParentheses` should not report the second expression as violation:
```ruby
a = []
a << (:foo == :bar ? :swag : :yolo)
```
Rubocop should not flag this as a style violation. Because the `<<` operator has precedence over the ternary operator and hence the result of the boolean expression is added to the array without the parentheses which makes the ternary operation a void expression yielding the `true`-option in every case.

## Actual behavior
Rubocop flags this as a style violation.

## Steps to reproduce the problem
Snippet above.

## RuboCop version
`0.79.0 (using Parser 2.7.0.2, running on ruby 2.6.3 x86_64-linux)`

**Comment by RST-J** [comment]
Sorry, this was a mistake, rubocop flags the inner parenthesis around the boolean expression.
